# .yaml config file
#
# The PID gains and controller settings must be saved in a yaml file that gets loaded
# to the param server via the roslaunch file (my_robot_control.launch).

ddr:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1500  #rate topic /ddr/joint_states

  # Effort Controllers ---------------------------------------
  leftWheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: left_wheel_hinge
    pid: { p: 10.0, i: 0.01, d: 0.1}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
  rightWheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: right_wheel_hinge
    pid: { p: 10.0, i: 0.01, d: 0.1}
    #pid: {p: 50.0, i: 0.1, d: 0.0}
